### PR TITLE
New Career section in Docs

### PR DESCRIPTION
**Pull request by johnlafleur**
## What
We will soon hiring and need to publish our open roles. 
This PR is about adding a Career section in the docs that details our hiring process in addition to our open positions.

### CODE REVIEW

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
     * [Marketo](integrations/sources/marketo.md)
     * [MySQL](integrations/sources/mysql.md)
     * [MSSQL](integrations/sources/mssql.md)
-    * [Microsoft SQL Server (MSSQL)](integrations/sources/mssql.md)
+    * [Microsoft SQL Server \(MSSQL\)](integrations/sources/mssql-1.md)
```
Comment (jrhizor) [inline docs/SUMMARY.md:25]: Wy are there two versions of the same doc here? `mssql.md` and `mssql-1.md`? SHoul

File: docs/career-and-open-positions/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+## **Our Recruiting Process**
+
+We will schedule 2 to 3 short calls with our founders and team. These will focus on two key areas:
+
+* Ensuring you will be a great cultural fit
+* Aptitude for the role
+
+If you succeed with the above, we will set you up with 1 or 2 more in-depth hour-long interviews to test how you actually think and work on some potentially real tasks for the position. It will also be an opportunity for you to see how we work. The team will be here to support you, as we want you to succeed. 
+
+Once all of this done, we will get back to you very soon. We tend to be very fast!
```
Comment (michel-tricot) [inline docs/career-and-open-positions/README.md:49]:
```suggestion
Once all of this done, we will discuss the process internally and get back to you very fast (velocity is everything here)!
```

File: docs/career-and-open-positions/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## **Our Benefits**
+
+* Unlimited permissionless vacation
```
Comment (michel-tricot) [inline docs/career-and-open-positions/README.md:53]:
```suggestion
* Unlimited vacation(s?)
```

File: docs/career-and-open-positions/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+* Health insurance: we let you choose the insurance of your choice and cover $500 for you + $200 for any dependents. This will evolve with time, especially after our Series A.
+* We'll give you a corporate card for expenses. Our philosophy is Freedom & Responsibiility. We trust you, just do what's best for the company.
+* We're strong believers in growth, as individuals and as a team. So anything that can help you grow can be expensed. For instance, free books
```
Comment (michel-tricot) [inline docs/career-and-open-positions/README.md:56]: is the sentence complete?

**Review by michel-tricot — APPROVED** [review]

File: docs/SUMMARY.md (hunk 1 above)
Comment (johnlafleur) [reply docs/SUMMARY.md]: Indeed! Fixed

**Pull request merged**